File: docs/java/tpv/src/model/Local.java
package model;

import java.util.List;

public class Local {
    private long idLocal;
    private String nombre;
    private String direccion;
    private List<Empleado> empleados;

    public long getIdLocal() {
        return idLocal;
    }

    public void setIdLocal(long idLocal) {
        this.idLocal = idLocal;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    public List<Empleado> getEmpleados() {
        return empleados;
    }

    public void setEmpleados(List<Empleado> empleados) {
        this.empleados = empleados;
    }

}

File: docs/cs/src/model/Local.cs
using System.Collections.Generic;

namespace Model
{
    public class Local
    {
        public long IdLocal { get; set; }
        public string Nombre { get; set; }
        public string Direccion { get; set; }
        public List<Empleado> Empleados { get; set; }
    }
}
